front/app/api/invites/useBulkInviteEmails.test.ts

```diff
@@ -0,0 +1,57 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+
+import useBulkInviteEmail from './useBulkInviteEmails';
```

Comment:
```suggestion
import useBulkInviteEmails from './useBulkInviteEmails';
```

Reply:
Same in the rest of the file, may need a search and replace.

Reply:
Good point, names should match 👍